pages/glossary/at-the-end.md

```diff
@@ -0,0 +1,26 @@
+---
+title: At the end of a block
```

Comment:
I think "after a block" is more appropriate here. "At the end" to me suggests it's the last node inside the block, not the first node outside of it.